### PR TITLE
Making some tweaks to the draft

### PR DESCRIPTION
**Pull request by tghosth**
This Pull Request relates to issue #1612

### CODE REVIEW

File: 5.0/en/0x53-V53-WebRTC.md [hunk 1]
```diff
@@ -22,17 +22,21 @@ It is important to note that these security requirements **do not apply to devel
...
 | :---: | :--- | :---: | :---: | :---: |
-| **53.1.1** | [ADDED] Verify that the TURN service allows access only to IP addresses that are not reserved for special purposes (e.g., internal networks, broadcast, loopback). Note that this applies to IPv4 and IPv6 addresses. | ✓ | ✓ | ✓ |
+| **53.1.1** | [ADDED] Verify that the TURN service only allows access to IP addresses that are not reserved for special purposes (e.g., internal networks, broadcast, loopback). Note that this applies to both IPv4 and IPv6 addresses. | ✓ | ✓ | ✓ |
 | **53.1.2** | [ADDED] Verify that the TURN service is not susceptible to resource exhaustion when legitimate users attempt to open a large number of ports on the TURN server. |  | ✓ | ✓ |
 
 ## V53.2 Media
 
 Media security is paramount in WebRTC applications, as it directly impacts the confidentiality, integrity and availability of audio and video communications. By addressing these security concerns, developers can safeguard the media streams in WebRTC applications, preventing eavesdropping, tampering, and denial-of-service attacks that could compromise user privacy and communication quality.
 
+These requirements only apply to systems that...
```
Comment (tghosth) [inline 5.0/en/0x53-V53-WebRTC.md:38]: @sandrogauci do you think we could add the circumstances here when this section will or will not be applicable?

File: 5.0/en/0x53-V53-WebRTC.md [hunk 2]
```diff
@@ -47,6 +51,8 @@ Media security is paramount in WebRTC applications, as it directly impacts the c
 
 Signalling is a critical component of WebRTC applications, responsible for coordinating communication sessions between peers. The security of the signalling process is essential to prevent unauthorized access, eavesdropping, and service disruptions.
 
+These requirements only apply to systems that...
```
Comment (tghosth) [inline 5.0/en/0x53-V53-WebRTC.md:54]: @sandrogauci again, do you think we could add the circumstances here when this section will or will not be applicable?

**Review by jmanico — APPROVED** [review]

File: 5.0/en/0x53-V53-WebRTC.md (hunk 2 above)
Comment (sandrogauci) [reply 5.0/en/0x53-V53-WebRTC.md:54]: In this case, I cannot imagine realistic cases where you are not using some form of signalling server and also using your own WebRTC infrastructure. That said, the design is modular and we haven't seen everything yet. So I'll add basic text to be more explicit.

File: 5.0/en/0x53-V53-WebRTC.md (hunk 1 above)
Comment (tghosth) [reply 5.0/en/0x53-V53-WebRTC.md:38]: Ok cool, can you make changes directly to this PR or do you need to make a PR on my PR @sandrogauci ?

File: 5.0/en/0x53-V53-WebRTC.md (hunk 2 above)
Comment (tghosth) [reply 5.0/en/0x53-V53-WebRTC.md:54]: Great, see comment above :)

**Pull request merged**